mod error;

use crate::error::Error;

#[tokio::main]
async fn main() -> Result<(), Error> {

    Ok(())
}
